python_bindings/setup.py

```diff
@@ -0,0 +1,53 @@
+from skbuild import setup, cmaker, utils
```

Comment:
Don't we need to have `skbuild` in pyproject.toml or requirements.txt or, uh, something?